R/stats-lm-tidiers.R

```diff
@@ -57,29 +57,31 @@
 #'   geom_abline(slope = seq(0, 3, by = 0.5), colour = "white") +
 #'   geom_smooth(se = FALSE) +
 #'   geom_point()
-#' 
+#'
 #' # column-wise models
 #' a <- matrix(rnorm(20), nrow = 10)
 #' b <- a + rnorm(length(a))
 #' result <- lm(b ~ a)
 #' tidy(result)
-#'
 #' @aliases lm_tidiers
 #' @export
 #' @seealso [tidy()], [stats::summary.lm()]
 #' @family lm tidiers
-tidy.lm <- function(x, conf.int = FALSE, conf.level = .95,
-                    exponentiate = FALSE, quick = FALSE, ...) {
+tidy.lm <- function(x,
+                    conf.int = FALSE,
+                    conf.level = .95,
+                    exponentiate = FALSE,
+                    quick = FALSE,
+                    ...) {
   if (quick) {
     co <- stats::coef(x)
-    if (inherits(x,'mlm')) {
-      ret <- data.frame(response = rep(colnames(co), each = nrow(co)),
-                        term = rep(rownames(co), times = ncol(co)),
-                        estimate = as.numeric(co), stringsAsFactors = FALSE)
-    } else {
-      ret <- data.frame(term = names(co), estimate = unname(co),
-                        stringsAsFactors = FALSE)
-    }
+
+    ret <- data.frame(
```

Comment:
Let's use `tibble` here instead of `data.frame(..., stringsAsFactors = FALSE)`

Reply:
This change didn't make it through?

Reply:
Huh. Sorry, missed that I had also changed `lm` tidier files.